ui/client/components/graph/NodeUtils.js

```diff
@@ -4,6 +4,10 @@ import ProcessUtils from '../../common/ProcessUtils.js'
 
 class NodeUtils {
 
+  isNode = (obj) => {
+    return !_.isEmpty(obj) && 'id' in obj && 'type' in obj
```

Comment:
You can use there _.has from lodash.

Reply:
Yup, I know. Can we leave it as it is or should I switch to lodash helper?

Reply:
I think it will be looks better with lodash.

Reply:
Done.